Client/Frontend/Widgets/TwoLineCell.swift

```diff
@@ -62,6 +65,35 @@ class TwoLineTableViewCell: UITableViewCell {
     }
 }
 
+class HistoryTableViewCell: TwoLineTableViewCell {
+    let _imageView = UIImageView()
+    let borderView = UIView()
+
+    // Override the default imageView so that borderView can add our own imageView into its subviews.
+    override var imageView: UIImageView? {
```

Comment:
Nit: What's the advantage of overriding `imageView` with `_imageView` instead of exposing a custom imageView named `iconImageView`?